src/bots/discord.ts

```diff
@@ -55,8 +55,11 @@ export default class DiscordBot extends BotClient {
   }
 
   public async getUser(): Promise<User> {
-    const userID = this.bot.user.id;
-    return new User(this, userID);
+    const userID = this.bot.user?.id;
+
+    if (!userID) this.logger.error('Bot user not found.');
```

Comment:
I'd support a linter rule that blocks these kind of statements, I believe brackets should be mandatory

Reply:
Is there a rule for that? I could not find one yet

Reply:
https://eslint.org/docs/rules/curly

Reply:
Ok, fixed